src/tests/end-to-end/tests/details-view/reflow-ui.test.ts

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { narrowModeThresholds } from 'DetailsView/components/narrow-mode-detector';
+import { Browser } from '../../common/browser';
+import { launchBrowser } from '../../common/browser-factory';
+import { detailsViewSelectors } from '../../common/element-identifiers/details-view-selectors';
+import { DetailsViewPage } from '../../common/page-controllers/details-view-page';
+import { scanForAccessibilityIssues } from '../../common/scan-for-accessibility-issues';
+
+describe('Details View -> Assessment -> Reflow', () => {
+    let browser: Browser;
+    let detailsViewPage: DetailsViewPage;
+    const height = 400;
+
+    beforeAll(async () => {
+        browser = await launchBrowser({
+            suppressFirstTimeDialog: true,
+            addExtraPermissionsToManifest: 'fake-activeTab',
+        });
+
+        detailsViewPage = (await browser.newAssessment()).detailsViewPage;
+    });
+
+    afterAll(async () => {
+        if (browser) {
+            await browser.close();
+            browser = undefined;
+        }
+    });
+
+    describe('With command bar collapsed', () => {
+        beforeEach(async () => {
+            const width = narrowModeThresholds.collapseCommandBarThreshold - 1;
+            await detailsViewPage.setViewport(width, height);
+            await detailsViewPage.waitForSelector(detailsViewSelectors.commandBarMenuButton);
+        });
+
+        it.each([true, false])(
+            `should pass accessibility validation with high contrast mode=%s`,
+            async highContrastMode => {
+                await browser.setHighContrastMode(highContrastMode);
```

Comment:
It helps with reliability to include an `await pageUnderTest.waitForHighContrastMode(highContrastMode);` after the set call (we should probably do some sort of separate refactor PR to encapsulate the repetitive pattern into some sort of test helper...)

Reply:
(same comment everywhere else in this file where `setHighContrastMode` is called)